system/lib/libc/musl/src/errno/__strerror.h

```diff
@@ -16,7 +16,7 @@ E(ENOENT,       "No such file or directory")
 E(ESRCH,        "No such process")
 E(EEXIST,       "File exists")
 
-E(EOVERFLOW,    "Value too large for data type")
+E(EOVERFLOW,    "Value too large for defined data type")
```

Comment:
Can you wrap these changes in `#if __EMSCRIPTEN__`, and add a comment that we are trying to match llvm-libc / glibc?